velox/functions/lib/MapFromEntries.cpp

```diff
@@ -30,7 +30,10 @@ static const char* kIndeterminateKeyErrorMessage =
     "map key cannot be indeterminate";
 static const char* kErrorMessageEntryNotNull = "map entry cannot be null";
 
-// See documentation at https://prestodb.io/docs/current/functions/map.html
+// allowNullEle If true, will return null if map has
+// an entry with null as key or map is null (Spark's behavior)
+// instead of throw execeptions(Presto's behavior)
+template <bool allowNullEle>
```

Comment:
Maybe use a more straight-forward name like `throwForNull` as it is the main difference.